pkg/virt-controller/services/net_binding.go

```diff
@@ -0,0 +1,30 @@
+package services
+
+import (
+	"fmt"
+
+	v1 "kubevirt.io/api/core/v1"
+
+	"kubevirt.io/kubevirt/pkg/hooks"
+)
+
+func BindingPluginSidecarList(vmi *v1.VirtualMachineInstance, config *v1.KubeVirtConfiguration) (hooks.HookSidecarList, error) {
+	var pluginSidecars hooks.HookSidecarList
+	for _, iface := range vmi.Spec.Domain.Devices.Interfaces {
+		if iface.Binding != nil {
+			var exist bool
+			var pluginInfo v1.InterfaceBindingPlugin
+			if config.NetworkConfiguration != nil && config.NetworkConfiguration.Binding != nil {
+				pluginInfo, exist = config.NetworkConfiguration.Binding[iface.Binding.Name]
+			}
+
+			if !exist {
```

Comment:
Please consider adding validation for adding binding plugin to Kubevirt CR, at least that the image is not empty.
Running pods with containers who has empty image should fail.

It may also reduce some error checking like this one if its validated at the API level.